joern-cli/frontends/rubysrc2cpg/src/main/scala/io/joern/rubysrc2cpg/astcreation/AstForExpressionsCreator.scala

```diff
@@ -114,6 +114,18 @@ trait AstForExpressionsCreator { this: AstCreator =>
     case ctx: ConcatenatedStringExpressionContext => Seq(astForConcatenatedStringExpressions(ctx))
   }
 
+  protected def astForRegexInterpolationPrimaryContext(ctx: RegexInterpolationContext): Seq[Ast] = {
```

Comment:
Please put a comment here explaining that regex interpolation has been reduced/modelled just as a set of statements here